### ISSUE TITLE
Discuss: shouldn't "XHR" and "script" columns be merged into one "script" column?

### ISSUE THREAD

**Issue by gorhill** [gorhill/httpswitchboard#83]
I've never been decided on this, so I just went with reflecting Chromium API.

I open this to welcome feedback so I can make my mind on this.

Points:
- For less geeky users, `XHR` is gibberish.
- Allowing `XHR` without `script` accomplish nothing.
- Allowing `script` without `XHR`... I need concrete scenarios where this is useful.
- `XHR` is nice for dynamically built and/or interactive web pages.
- `XHR` sucks if it is used to send tracking or analytic data without user's informed consent.
- But then, tracking or analytics is easily blocked by blacklisting some hostnames obviously dedicated to that purpose.
- But then, what about less obvious hostnames... Happens often? (is that a valid concern, if someone whitelisted `script` on that hostname...)
- Merging would result in a nice gain in number of pixels available horizontally, can be used to bump up the width for hostname cells in order reduce occurrences of truncation of the hostname.
- Mitigation: users can find out XHR requests made to a server using _Statistics_ page.

I enumerated all this, I still don't have an opinion.
